core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/dot-edit-content-category-field.component.ts

```diff
@@ -74,13 +90,21 @@ export class DotEditContentCategoryFieldComponent {
      *
      * @returns {void}
      */
-    showCategories(): void {
-        this.#dialogService.open(DotEditContentCategoryFieldDialogComponent, {
-            showHeader: false,
-            styleClass: 'category-field__dialog',
-            width: '1000px',
-            height: '600px',
-            position: 'center'
-        });
+    showCategoriesSidebar(): void {
+        this.disableSelectCategoriesButton.set(true);
+        this.#componentRef = this.sidebarHost.viewContainerRef.createComponent(
+            DotEditContentCategoryFieldSidebarComponent
+        );
+
+        this.setSidebarListener();
+    }
+
+    private setSidebarListener() {
+        this.#componentRef.instance.closedSidebar
+            .pipe(takeUntilDestroyed(this.#destroyRef), delay(CLOSE_SIDEBAR_CSS_DELAY_MS))
```

Comment:
Why do we need a delay?

Reply:
Because the sidebar has a CSS transition of 150ms making the slide effect hide. If I remove the component the sidebar only disappears without any slide.